jre_emul/android/platform/libcore/luni/src/test/java/libcore/java/net/URLConnectionTest.java

```diff
@@ -2120,6 +2125,33 @@ private void testInputStreamAvailable(TransferKind transferKind) throws IOExcept
 //    }
 
     /**
+     * Test that we can inspect the SSL session after connect().
+     * http://code.google.com/p/android/issues/detail?id=24431
+     */
+    public void testInspectSslAfterConnect() throws Exception {
```

Comment:
We can't accept this as part of our test suite since all tests need to run without making any external http connections (even to Google!) for security reasons. 

However, it is a useful example, so please move this method into the main method of a new class in jre_emul/misc-tests (such as jre_emul/misc-tests/InspectSSLAfterConnect.java).